plugins/nodes/src/js/pages/NodesOverview.js

```diff
@@ -204,22 +203,6 @@ var NodesOverview = React.createClass({
     );
   },
 
-  getFilterInputText() {
```

Comment:
I don't understand what this change has to do with the rest? How does this contribute to having tabs for master and agent?

Reply:
Next time, please do a separate commit for that